api/python/notebooks/analysis_demo/comp_bio_query_data_and_metadata.ipynb

```diff
@@ -0,0 +1,1056 @@
+   "metadata": {},
+   "source": [
+    "With the exception of `soma_joinid` these columns are defined in the [Cell Census schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema_0.0.1.md#feature-metadata--census_objcensus_dataorganismmsrnavar--somadataframe). Similarly to the cell metadata, we can use the same operations to learn and fetch gene metadata.\n",
+    "\n",
+    "For example, to get the `feature_name` and `feature_length` of the genes `\"ENSG00000161798\"` and `\"ENSG00000188229\"` we can do the following."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>feature_name</th>\n",
+       "      <th>feature_length</th>\n",
+       "      <th>feature_id</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>AQP5</td>\n",
+       "      <td>1884</td>\n",
+       "      <td>ENSG00000161798</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>TUBB4B</td>\n",
+       "      <td>2037</td>\n",
+       "      <td>ENSG00000188229</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  feature_name  feature_length       feature_id\n",
+       "0         AQP5            1884  ENSG00000161798\n",
+       "1       TUBB4B            2037  ENSG00000188229"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gene_metadata = census[\"census_data\"][\"homo_sapiens\"].ms[\"RNA\"].var.read_as_pandas_all(\n",
+    "    value_filter = \"feature_id in ['ENSG00000161798', 'ENSG00000188229']\",\n",
+    "    column_names = [\"feature_name\", \"feature_length\"],\n",
+    ")\n",
+    "\n",
+    "gene_metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Querying expression data\n",
+    "\n",
+    "The most convenient way to query and fetch expression data is to us the `get_anndata` method of the `cell_census` API. This is a powerful method that combines the column selection and value filtering we described above to obtain slices of the expression data based on metadata queries.\n",
```

Comment:
Re: "... most convenient ...":  I don't agree with this statement - I think we should be educating people on how to use the underlying expression query, as it is far more powerful and can handle a wider set of use cases (and often will have better performance).  That said, I recognize that using `get_anndata` for this notebook is appropriate - I'm just not loving the 'most convenient" phrase....

Reply:
Agreed I'll changed it to "a convenient"